### PR TITLE
Replaced a lot of Magento logos with OpenMage ones

### PR DESCRIPTION
**Pull request by fballiano**
This repo has a lot of Magento logo images and some old OpenMage logos (different from the one on the new websites). This PR is trying to fix this situation as much as possibile:
- replaced "/errors" pages logos with the new svg version
- replaced "installation" logos with the new official version
- replaced adminhtml themes logos with the new official version (sometimes with the svg, sometimes keeping the original formats for backward compatibility)
- replaced all favicons using the new official one
- replaced RWD theme logos with the new official version (keeping the original formats for backward compatibility)

I know it is a big PR but this project deserves to have a coherent image.

Also, because magento 1 didn't have any support for "2x" images (at least in logos) we should really migrate everything possible to SVG, but that would be for 20.x only, thus I left any unsave SVG conversion out of this PR, thus I think it's safe to merge it on the 1.9 branch.

Some screenshots:
Schermata 2021-05-05 alle 23 43 43
Schermata 2021-05-05 alle 23 37 24
Schermata 2021-05-05 alle 23 00 30
Schermata 2021-05-05 alle 23 00 28

### Fixed Issues (if relevant)
1. Fixes https://github.com/OpenMage/magento-lts/issues/1589

### CODE REVIEW

**Review by Flyingmana — CHANGES_REQUESTED** [review]
While I may not be objective here, as I was introducing the logo you replace here, I do veto for a reason.
This is a big change in Branding. When I did introduce the Logo, I tried to get as many opinions and votes as possible. There were two alternatives back then offered. So it was a community decision back then, and if we do change the branding everywhere with the new logo, we definitely should make this a community decision, not a maintainer decision.

**Comment by fballiano** [comment]
but then why the website has a different logo? which one is the real one?
and it's it better to keep having multiple magento logos after all this
time?

Il Gio 6 Mag 2021, 06:07 Daniel Fahlke ***@***.***> ha
scritto:

**Comment by Flyingmana** [comment]
@fballiano  which of the possible answers would speak against making this a community decision?

**Comment by fballiano** [comment]
it should be a community decision, I love community decisions.

but at this moment the non coherent branding is a bad thing for the project.

I also have to say that the new logo is already in the source code and the website, just not everywhere.

**Comment by ADDISON74** [comment]
My opinion:

1) OpenMage needs a consistency of logos used in both Frontend and Backend. Also in default themes. All Magento logos need to be replaced. In Frontend I see it in in error report pages, in Backend I it when logging in and at the top left.

2) The idea of using SVG is not bad at all. If I don't like the color, I can make an immediate change without taking the image to an editor and compressing it, which will definitely not good looking. The fact that the logo is SVG will make it easier to replace it in the future.

3) If I could consider the first two mandatory, what the community can determine is whether OpenMage needs a new logo that conveys "something." The chosen color, the text, are vital. Colors like green or blue are preferred because they represent health and tranquility. It can be purple too. I do not agree using gradients.

**Comment by ADDISON74** [comment]
As you know there is a theme for Backend that is set by default. Whoever wants the Magento theme can set it in Settings. If a new color is set for the OpenMage logo then this color will be the main one. All the colors used in the theme must be analyzed with a tool like this one: https://color.adobe.com/create/color-wheel. Thus we create a beneficial harmony for our eyes, being a pleasure to stay in the interface sometimes for a longer time.

**Comment by colinmollenhour** [comment]
Thanks for doing all of the lifting on this @fballiano!

I think everyone agrees on replacing the Magento logo, so that's good. 👍 

I suppose when we redesigned the public site and introduced the new logo that caused this confusion so I apologize for that. The intent was to get a professional designer to give the public site a face-lift keeping the spirit of the original intact to whatever degree was appropriate. I/we gave the designer a lot of freedom in this and I think the result was fantastic. The community also only had great things to say about the new design when it was previewed before it was merged so I thought there was already more or less a consensus on the "new" logo more or less replacing the "old" logo? I never set out to "change" the logo but effectively I suppose we did and although it is different it is also quite similar, with dropping the gradient for a solid color being the only major difference (to my eyes). Kudos to @Flyingmana for designing a logo that was able to survive a complete site redesign with minimal changes and still look great!

I personally am not opposed to keeping the old logo in the original admin theme and having the new one in the new theme, it's not too uncommon for old designs to linger in some places. Neither logo looks good with the old Magento admin theme's pale green and orange tones.. 😄 

But I can certainly see the argument for having one consistent brand to present everywhere and so would agree with going ahead and replacing it with the new one as Fabrizio has done with this pull request. So +1 for merging from me.

**Comment by Flyingmana** [comment]
One correction, I did not design the logo, I did had a Designer friend commissioned for one.
So the Kudos belong to Nirit Nagar https://www.xing.com/profile/Nirit_Nagar

**Comment by fballiano** [comment]
just a couple of notes:
- the new logo is awesome
- although it seems weird to see in on the legacy adminhtml theme, it's more readable/visible than the old but most of all I'm not comfortable enabling the new default adminhtml for my customers, it's a nice theme but the text/background contrast and the color scheme makes it not easy to read while the legacy theme (although not really modern looking) doesn't have this problem (the backend IMHO has to be really usable, form over design for sure). sadly I've no competence to improve the graphic design of the new adminhtml theme.

**Comment by ADDISON74** [comment]
If it were up to me I would make the theme for Backend a separate project. Thus, people will be found to contribute ideas and improve it, including a specially chosen color palette, fonts, etc. OM will be left with code changes so that those who want to install the theme to use it instantly. I appreciate the effort for this theme but all the users in the installations I manage have asked me for the original Magento version.

File: errors/default/page.phtml
Comment (luigifab) [inline errors/default/page.phtml:45]: " are missing for height attribute :)

File: skin/adminhtml/default/default/boxes.css
Comment (luigifab) [inline skin/adminhtml/default/default/boxes.css:725]: instead of using a second svg file, there is: `filter:grayscale(100%)`

File: errors/default/images/logo.svg
Comment (fballiano) [reply errors/default/images/logo.svg:9]: they're used and can't be removed, I didn't create the SVG but anyway I woulnd't make them inline cause it's easier to change them if needed

Comment (fballiano) [reply errors/default/images/logo.svg:9]: Schermata 2021-05-11 alle 14 10 07

removing those class...

File: errors/default/images/logo.svg
Comment (fballiano) [reply errors/default/images/logo.svg]: done in https://github.com/OpenMage/magento-lts/pull/1599/commits/b01910e7ee5de9725194bd8dcea85b20e8e663a8

File: errors/default/page.phtml
Comment (fballiano) [reply errors/default/page.phtml]: done, although I think this note is completely useless, quotes should be added when necessary, most of all if you're so worried about removing "x=0" from the svg, then we can reduce the filesize removing unnecesary quotes.

**Comment by fballiano** [comment]
Admins do we have to do to move this forward?

**Review by kkrieger85 — APPROVED** [review]

**Comment by fballiano** [comment]
is the actual one:
Schermata 2021-07-04 alle 14 47 16

really considered better than the SVG one? The OM symbol it's ok but the "Admin Panel" font looks awful, please let's get rid of it!

**Comment by ADDISON74** [comment]
@fballiano - I totally agree. The text doesn't look good at all. It seems the letters spacing is different and also the font used. I am wondering if "Admin Panel" is mandatory as we all know this is OM Backend. Log + OpenMage should enough.

**Review by elidrissidev — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Review by luigifab — APPROVED** [review]

**Review by tmewes — APPROVED** [review]

**Review by kiatng — APPROVED** [review]

**Review by m-overlund — APPROVED** [review]

**Comment by fballiano** [comment]
Hi @Flyingmana, how could we move this forward?
We've 7 positive reviews on this PR (the most reviewed open PR at the moment) but you were talking about community vote. Although I think the new logo is just a better version of the old one I see your point, but how can we do that? Also, how could we measure if a "vote" is actually successful?

**Comment by colinmollenhour** [comment]
I think @Flyingmana's change request was the only hold-up but I'm not sure if he is still objecting or just didn't dismiss his change request..

> we definitely should make this a community decision, not a maintainer decision

It appears to me the community has spoken as there are no arguments made against the change (Daniel's objection seems to be more a procedural one and not a substantive one) and many approvals.

**Comment by luigifab** [comment]
Not sure, but it seems that old emails sent appear with a bad logo:

image
image

This is only my history?

**Comment by ADDISON74** [comment]
@luigifab - Most likely the template used displays the current image on the server not the old one.

**Review by S0FTWEX — COMMENTED** [review]
Changes in file skin/adminhtml/default/default/boxes.css (#1599) was not merged into 20.0.15

**Comment by fballiano** [comment]
@Flyingmana it really seems that @S0FTWEX is right, on branch 20.0 skin/adminhtml/default/default/boxes.css doesn't reflect the changes from this PR. how could we solve this?

**Pull request merged**
